nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/queue/clustered/TestContentRepositoryFlowFileAccess.java

```diff
@@ -121,7 +121,7 @@ public void testEOFExceptionIfNotEnoughData() throws IOException {
 
         try {
             repoStream.read();
-            Assert.fail("Expected EOFException because not enough bytes were in the InputStream for the FlowFile");
+            fail("Expected EOFException because not enough bytes were in the InputStream for the FlowFile");
         } catch (final EOFException eof) {
             // expected
         }
```

Comment:
It looks like this can be changed to `assertThrows(EOFException.class, () -> repoStream.read());`